hack/build-acceptable-bundles.sh

```diff
@@ -3,10 +3,27 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+
+# Function to remove the sha and digest from the image name
+# from: quay.io/konflux-ci/task1:0.1-1234@sha256:5678 to quay.io/konflux-ci/task1:0.1
+strip_image_tag() {
+  echo "$1" | sed 's/\(:[^-]*\).*/\1/'
```

Comment:
I learned this recently:

```suggestion
  sed 's/\(:[^-]*\).*/\1/' <<< "$1"
```

Reply:
That's cool. Updated.